### ISSUE TITLE
Minor issues in the midazolam report

### ISSUE THREAD

**Issue by JanSchlender** [Open-Systems-Pharmacology/Midazolam-Model#13]
[midazolam_evaluation_report_JS.pdf](https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library/files/4759751/midazolam_evaluation_report_JS.pdf)

**Comment by sfrechen** [comment]
@JanSchlender 
Could you create respective issues in https://github.com/Open-Systems-Pharmacology/Midazolam-Model. The raised points are valid "feature" wishes.
The point regarding figure captions is rather a common suggestion then midazolam specific.

We can then adress it for the next version.

**Comment by JanSchlender** [comment]
Agree, that the data comparision/consistency point is a feature wish
Why are figure captions or plots with a lin-scale for concentrations a feature in a midazolam report while present in other reports?

**Comment by sfrechen** [comment]
Well, it could technically be implemented. But it would be an additional "feature" of the report.
